### ISSUE TITLE
IBM PCjr: Joysticks drift to the right

### ISSUE THREAD

**Issue by einstein95** [86Box/86Box#3140]
**Describe the bug**
Instead of a range of 0-100, the PCjr in 86box sees a range of 3-113, with a neutral of 60, which makes games like Demons Attack move the player to the right.

**To Reproduce**
1. Boot IBM PCjr to BASIC
2. Run the following script:
```basic
10 PRINT STICK(0);STICK(1)
20 GOTO 10
```
4. Move the joystick along the axes and watch the value change

**Expected behavior**
The range should be from 0 to 100 as described in the tech docs.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - 86Box version: build 4455
 - Build information: old dynarec, x64, dynamic

**Additional context**
The way I see it, there should be more options when connecting a controller w/r/t joystick sensitivity and dead zones. Using an Xbone S controller and I can see the axis data changing without me even moving the stick.

Checking with the IBM XT and it sees a more reasonable neutral hovering just above 50, but also going over 100. I haven't checked the tech doc to see what its range should be, but it's likely that it too is 0-100.

**Comment by FavoritoHJS** [comment]
Ah, the joys of analog joysticks...

The value read is the amount of loops of a piece of code it takes to empty a capacitor. If the CPU is a different speed, the amount of loops is different as well.

If the game has joystick calibration options, use those. Otherwise, I guess a "sensitivity" option could be added to the joystick configuration options? Though I wonder if that will break stuff.

**Comment by einstein95** [comment]
Yeah, I would use in-game options, except these predate that sort of thing and just assumed that the values wouldn't need adjusting (this is IBM, when they were doing tricks that Apple would continue)

**Comment by OBattler** [comment]
See if this still happens.

**Comment by einstein95** [comment]
> See if this still happens.

Still occurs in a self-compiled build of 86Box v4.0 [x86_64, old dynarec] commit f1174247f
